ruff.toml

```diff
@@ -0,0 +1,17 @@
+# Enable flake8-bandit (`S`) and isort (`I`) rules.
+select = ["E", "F", "S", "I"]
```

Comment:
Can you quicky just doc that "E" is flake8's error and "F" is pyflakes

Reply:
Added.